### DISCUSSION TITLE
python背景和相關的基礎知識

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#373] [General]
這幾篇文章介紹了一些有關python的背景、版本歷史、目前的應用等等。對於不太確定自己學習Python這個程式語言將來對自己有甚麼樣的幫助的人來說可以也很好的幫助👍
[https://zh-tw.coderbridge.com/series/36b9cbc8faea4c50a59b7e8783c14b99/posts/0f278f41b8294d85a726ef739ace5225](https://zh-tw.coderbridge.com/series/36b9cbc8faea4c50a59b7e8783c14b99/posts/0f278f41b8294d85a726ef739ace5225)
[https://www.biaodianfu.com/python-intro.html](https://www.biaodianfu.com/python-intro.html)

**Comment by handsomeniko** [comment]
推推，非常有幫助~~

**Comment by xkllkx** [comment]
謝謝分享~對python更了解了

**Comment by chupochun** [comment]
感謝分享 讓我對python更有興趣了

**Comment by pekoranoinu** [comment]
感恩分享 對學習python更有動力了

**Comment by ThomasMeng0515** [comment]
感恩分享 對學習python更有想法了

**Comment by lx3211** [comment]
謝謝分享～
